### DISCUSSION TITLE
Possibility to create a rule depending on the body text not body path

### DISCUSSION THREAD

**Discussion by bykof** [mockoon/mockoon#405] [Help] [answered]
Hi folks, 

is it possible in mockoon to create a rule depending only on the body text not a json or form path?

I receive a xml request and want to define a rule depending on an id inside this xml. 

Thank you very much for your help!

**Comment by 255kb** [comment, marked as answer]
Hi @bykof 
As you saw already, this is not yet possible but there is already an opened issue: https://github.com/mockoon/mockoon/issues/271
Sorry for the late reply 😃

**Reply by bykof** [reply]
Yup that's why I created a PR a minute ago :D but thank you anyway!
